### ISSUE TITLE
Decision tree with split training and test set

### ISSUE THREAD

**Issue by arlyon** [arlyon/dmml#29]
> Repeat steps 3 and 4, this time using training and testing data sets instead of the cross validation. That is, build the classifier using the training data set, and test the classifier using the test data set. Note the accuracy.

Randomly partition into two groups, train on one half, test on other half.
